mabl snippets/randomDropdownSelector.js

```diff
@@ -0,0 +1,12 @@
+function mablJavaScriptStep(mablInputs, callback) {
+
+  const myfunction = ()=> {
+    let select = document.getElementById('day'); //Replce 'day' with your CSS selector
+    let items = select.getElementsByTagName('option'); //Confirm your element tagnames and adjust as necessary
+    let index = Math.floor(Math.random() * items.length);
```

Comment:
I could be wrong, but I also think we want a `-1` at the end since the array would be 0-indexed

Reply:
Works as is.